<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>690</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>690</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>630</width>
    <height>690</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_pubback">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>651</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    background-image: url(&quot;/nas/Viper/minseo/forui/pubback2.png&quot;);
    background-repeat: no-repeat;  /* 반복 방지 */
    background-position: center;  /* 중앙 정렬 */
    background-size: contain;  /* QLabel 크기에 맞게 비율 유지 */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_publish">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>530</y>
      <width>501</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Myriad Pro</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: none;
    background: url(&quot;/nas/Viper/minseo/forui/pubbutton.png&quot;) no-repeat center;
    background-size: contain;
	border-radius: 10px;
	color: white;
	font-size: 19px;
}

QPushButton:pressed {
    background: url(&quot;/nas/Viper/minseo/forui/pubbutton.png&quot;) no-repeat center;
    background-size: contain;
}
</string>
    </property>
    <property name="text">
     <string>Publish</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Myriad Pro</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {

    background-repeat: no-repeat;  /* 반복 방지 */
    background-position: center;  /* 중앙 정렬 */
    background-size: contain;  /* 크기를 자동 조정 */
    color: white;  /* 글자 색상 (흰색) */
    font-size: 8px;  /* 글자 크기 */
  
    text-align: center;  /* 텍스트 중앙 정렬 */
}</string>
    </property>
    <property name="text">
     <string>Publish to ShotGrid</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_thumbnail">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>239</width>
      <height>150</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>150</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_memo">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>239</width>
      <height>150</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>150</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #242426;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_checkbox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>410</y>
      <width>239</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>100</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_publish_info">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>291</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #242426;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="margin">
     <number>20</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
